content/blog/getting-started-with-private-cloud-business-edition-apis.md

```diff
@@ -0,0 +1,365 @@
+8. Select the Copy icon to save your information to your desktop.
+9. Click the Close button to continue. You are returned to the main API page, where you can now generate the access token using a sample code provided.
+
+Example code:
+
+```python
+from oauthlib.oauth2 import BackendApplicationClient         
+from requests.auth import HTTPBasicAuth         
+from requests_oauthlib import OAuth2Session         
+
+client = BackendApplicationClient('e21f3028-8097-4a4f-b491-a49b1d102d4a')         
+        
+oauth = OAuth2Session(client=client)         
+auth = HTTPBasicAuth('e21f3028-8097-4a4f-b491-a49b1d102d4a', '05656940014f11ef9946a2408e898685')         
+   
+token = oauth.fetch_token(token_url='https://sso.common.cloud.hpe.com/as/token.oauth2', auth=auth)         
+print(token["access_token"])
+```
+
+> *Note*: Make sure the modules oauthlib, requests and requests_oauthlib are installed in your environment, if not just install these using:
+
+```powershell
+pip install oauthlib
+pip install requests
+pip install requests_oauthlib
+```
+
+When running the example code JSON formatted data will be returned, like this:
+
+```json
+{
+    "access_token": "eyJhbGciOiJSUzI1NiIsImtpZCI6IjJGSmhvZ1lRMDZNazNBc2Q4UU8zU09ZVE9wayIsInBpLmF0bSI6ImRlejAifQ.eyJjbGllbnRfaWQiOiJlMjFmMzAyOC04MDk3LTRhNGYtYjQ5MS1hNDliMWQxMDJkNGEiLCJpc3MiOiJodHRwczovL3Nzby5jb21tb24uY2xvdWQuaHBlLmNvbSIsImF1ZCI6ImV4dGVybmFsX2FwaSIsInN1YiI6Im1hcmsudmFuLnNpbGZob3V0QGhwZS5jb20iLCJ1c2VyX2N0eCI6ImVmMDc3MDVjOGVhNjExZWNiNzNiMWEyNTZjMDNiNTQ2IiwiYXV0aF9zb3VyY2UiOiJjY3NfdG9rZW5fbWFuYWdlbWVudCIsInBsYXRmb3JtX2N1c3RvbWVyX2lkIjoiZDI2YmFmMjY4ZWE2MTFlYzljZmUwMmMwMzQ1YzI0M2MiLCJpYXQiOjE3MTM4NjI4OTYsImFwcGxpY2F0aW9uX2luc3RhbmNlX2lkIjoiZWYwMTA4MjMtYjMwMS00MWJmLTk0OWMtYTQ3NzE0OTRiMzg1IiwiZXhwIjoxNzEzODcwMDk2fQ.arTNjVsVZ-wcW5Ic5fuGUvBKAupzWvRokuvdzlW1I2UqXFoqp0-jw1cHVr-QgDUlBPXG6uc-aILnYXWN_h-QWOQQu1c8aTbLaqpfXEL89MndPyErF0x4By21JLoR1mq-8zkMEEJ2CHGOeBYau_hBim-SBr1BtcetX3BcFl4GoGRGS5lzL1nbwkC-Hi_OOs_2UnDH4ajyyPsp_Ka4wmsgHn1aSQhJjDFbxx4WiLmRdp8aNZT5r250v5EWR9qVeqE4TDOfKAf_BBhC2RB00mWAt1F_Rd4rPmMyZm0uPZd5f71aDjd3I5tl0gh-W2Z4HHt0jRKV-L1d4o52jzYkhL1nTA",
+    "expires_at": 1713870095.208547,
+    "expires_in": 7199,
+    "token_type": "Bearer"
+}
+```
+
+the JSON data holds the access_token itself and also the expiration time, which is 2 hours (7200 seconds).
+
+> *Note*: For simplicity and demo reasons we store the client_id and client_secret in the code, obviously that should never be done in production! How to implement a more secure method will be explained in the next episode of this series.
+
+## Calling an API
+
+A list of available APIs for HPE GreenLake for Private Cloud Business Edition can be found at <https://developer.greenlake.hpe.com/docs/greenlake/services/private-cloud-business/public/> When publishing this blog the APIs were still in beta, however the GA release is planned shortly after.
+
+In this example we will collect an overview of all System Software Catalogs.
```

Comment:
we --> you:

In this example you will collect an overview of all _System Software Catalogs_.